docs/source/getting_started/benchmarks/imagenette_benchmark.py

```diff
@@ -96,7 +105,7 @@
     transform=test_transforms
 )
 
-def get_data_loaders(batch_size: int):
+def get_data_loaders(batch_size: int, multi_crops: bool = False):
```

Comment:
I would make this a little more flexible by adding the argument collate_fun with a default value. Then `get_data_loaders` will support any collate function. Furthermore, the variable `swav_collate_fn` would not need to be defined this globally anymore, Instead, it could be defined e.g. in a helper model of the SwavModel.

Reply:
Not sure about this. This should just be a very simple code for the purpose of benchmarking. I'd prefer to keep it very "minimal".

Reply:
Let's leave it then - ok? We can do a follow-up issue. At some point we'll have to refactor the benchmarks a little bit anyways.